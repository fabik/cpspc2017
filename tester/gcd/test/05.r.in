15
10 24 10 21 3 25 11 15 13 29 10 11 29 20 24
