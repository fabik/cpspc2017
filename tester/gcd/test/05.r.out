1
30
59
84
108
132
153
173
188
201
212
224
235
245
255
